docs/getting-started/integrations/spring-boot.md

```diff
@@ -0,0 +1,165 @@
+---
+category: self-hosted
+meta-title: Compatibility with Spring Boot using a ZIP archive | CKEditor 5 Documentation
+meta-description: Integrate CKEditor 5 with Spring Boot using a ZIP archive.
+order: 130
+menu-title: Spring Boot
+---
+
+# Compatibility with Spring Boot using ZIP
+
+As a pure JavaScript/TypeScript application, CKEditor&nbsp;5 will work inside any environment that supports such components. While we do not offer official integrations for any non-JavaScript frameworks, you can include a custom configuration of CKEditor&nbsp;5 in a non-JS framework of your choice, for example, the Java-based [Spring Boot](https://spring.io/projects/spring-boot).
+
+{@snippet getting-started/use-builder}
+
+## Setting up the project
+
+This guide assumes you already have a Spring Boot project. You can create a basic Spring Boot project using [Spring Initializr](https://start.spring.io/). Refer to the [Spring Boot documentation](https://docs.spring.io/spring-boot/docs/current/reference/html/getting-started.html) to learn how to set up a project in this framework.
+
+This guide is using the "Spring Web" and "Thymeleaf" dependencies selected in the Spring Initializr. Here is the list of dependencies used in the demo project:
+
+```xml
+<dependencies>
+	<dependency>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-web</artifactId>
+	</dependency>
+	<dependency>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-test</artifactId>
+		<scope>test</scope>
+	</dependency>
+	<dependency>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starter-thymeleaf</artifactId>
+	</dependency>
+</dependencies>
+```
+
+## Integrating using ZIP
+
+<info-box info>
+	Our new CKEditor&nbsp;5 Builder does not provide ZIP output yet &ndash; but it will in the future. In the meantime, you can use one of the generic ZIP packages provided [on the download page](https://ckeditor.com/ckeditor-5/download/#zip).
+</info-box>
```

Comment:
Actually, it's not true anymore. You can download a zip from the builder, so we should mention that option here.

![image](https://github.com/user-attachments/assets/c04f69ad-c991-4e38-90d9-98a31a7051b4)

Reply:
We may simply remove this infobox, Builder is linked on top. I'll fix these.